wrapanapi/msazure.py

```diff
@@ -305,7 +305,14 @@ def current_ip_address(self, vm_name, resource_group=None):
                                      vm_name=vm_name)
         first_vm_if = vm.network_profile.network_interfaces[0]
```

Comment:
**SUGGESTION:** We have to go through a lot of steps to get to the IP address.   It would be helpful I think to add a comment to explain the process from a high level.  I can follow all the individual steps but if in anyway we could discuss why we have to go through so many to get this information in the comment I think that would  helpful to any maintainer.